python/python_direct/bindings.h

```diff
@@ -33,4 +33,7 @@ void bindMultiDevice(py::module& nvfuser);
 // Translate a CPP Fusion to a bindings python function
 std::string translateFusion(Fusion* f);
 
+// Add bindings for Cutlass GEMM Operations
+void bindCutlass(py::module& nvfuser);
```

Comment:
We should have this guarded by `NVFUSER_ENABLE_CUTLASS`.

Do we have a test on building with disabled `NVFUSER_ENABLE_CUTLASS`?

Reply:
> Do we have a test on building with disabled NVFUSER_ENABLE_CUTLASS?

I thought the build, A100, and H100 tests would fail if this wasn't correct.
This Github CI build also disables CUTLASS.